examples/DynaFlow/WSCC9/WSCC9_DisconnectLine/WSCC9.iidm

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com) and UPC/Citcea (https://www.citcea.upc.edu/)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<iidm:network xmlns:iidm="http://www.itesla_project.eu/schema/iidm/1_0" id="IEEE9" caseDate="2021-05-14T08:59:25.732+02:00" forecastDistance="0" sourceFormat="PSS/E">
+    <iidm:substation id="S5">
+        <iidm:voltageLevel id="VL1" nominalV="16.5" topologyKind="BUS_BREAKER">
+            <iidm:busBreakerTopology>
+                <iidm:bus id="B1" name="BUS1        " v="16.5" angle="0.0"/>
```

Comment:
Could it be "BUS1" instead of "BUS1        " ? (same question for all buses)

Reply:
Inherited from a direct PSS/E -> IIDM conversion in PowSyBl